src/components/FilterBar/FilterDisplay/Inputs/RelativeDateRangeInput.react.js

```diff
@@ -0,0 +1,208 @@
+import { RangeInput } from "./RangeInput.react";
+import { DateInput } from "./DateInput.react";
+export class RelativeDateRangeInput extends RangeInput {
+  constructor(props) {
+    super(props)
+    this.state = {
+      value: this.props.value || { from: null, to: null },
+      operator: this.props.value.operator || "absolute"
+    }
+  }
+
+  isAbsolute() {
+    return this.state.operator == "absolute"
+  }
+
+  intToMoment(value) {
+    if (Number.isNaN(value) || value == null) {
+      return null
+    }
+
+    if (value < 0) {
+      return moment().subtract(Math.abs(value), 'day')
+    } else {
+      return moment().add(Math.abs(value), 'day')
```

Comment:
I imagine you don't need `Math.abs` in this case, as you've already confirmed the value is positive by this point. No real harm in leaving it there.